### PR TITLE
jx test

### PR DESCRIPTION
**Pull request by whatever555**

### CODE REVIEW

**Comment by darajava** [comment]
:-1:

**Comment by whatever555** [comment]
👍

File: Jenkinsfile [hunk 1]
```diff
@@ -1,20 +1,55 @@
...
+       }
+
+       stage('Build'){
+         env.NODE_ENV = "production"
+         print "Environment will be : ${env.NODE_ENV}"
+         sh 'node -v'
+         sh 'npm install yarn'
+         //sh 'npm prune'
+         sh 'yarn install'
+
```
Comment (darajava) [inline Jenkinsfile]: unnecessary whitespace

File: Jenkinsfile [hunk 2]
```diff
@@ -1,20 +1,55 @@
...
+        echo "Deployment active. please stand clear"
+         //echo 'Push to Repo'
+         //sh './dockerPushToRepo.sh'
+
+         //echo 'ssh to web server and tell it to pull new image'
+         //sh 'ssh deploy@xxxxx.xxxxx.com running/xxxxxxx/dockerRun.sh'
+
+       }
+
+       stage('Cleanup'){
```
Comment (darajava) [inline Jenkinsfile]: space before `{`

**Comment by darajava** [comment]
Just a general comment: Will you please start leaving more meaningful commit messages and give a description on what the PR is for? `jenkins-test` is not very descriptive as a branch name either. A shower once in a while wouldn't hurt either.

**Comment by darajava** [comment]
I'm a collaborator you fuck
